### PR TITLE
People: List Invited Users

### PR DESCRIPTION
**Pull request by jeremysawesome**
We'd like to list the currently invited users to this blog via the Calypso interface. Currently this functionality exists in WP-Admin but not in Calypso.

The goal of this PR will be to:
1. List any invited users
2. (Potentially, API Allowing) Allow invitations to be removed.

The initial implementation will simply request and list all invited users. We can add pagination or infinite loading in the future as needed.

This is related to: https://github.com/Automattic/wp-calypso/issues/4426

### CODE REVIEW

File: client/my-sites/people/people-section-nav/index.jsx [hunk 1]
```diff
@@ -111,13 +111,18 @@ class PeopleSectionNav extends Component {
 				path: '/people/viewers/' + siteFilter,
 				id: 'viewers',
 			},
+			{
+				title: translate( 'Invitations', { context: 'Filter label for invitations list' } ),
```
Comment (a8ci18n) [inline client/my-sites/people/people-section-nav/index.jsx]: Hi! I've found a possible matching string that has already been translated **9** times:
`translate( 'Invitations', { context: 'Filter label for people list'} )` *ES Score: 24*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/4426_implement_invitees_list)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?

**Comment by apeatling** [comment]
@jeremysawesome Haven't seen any activity on this one for a while, what's the status?

**Comment by jeremysawesome** [comment]
> @jeremysawesome Haven't seen any activity on this one for a while, what's the status?

Hi @apeatling thanks for checking up. Unfortunately my priorities have changed and I can no longer give this PR and ticket the love that it deserves. Feel free to have someone pick up the branch and run with it 👍

**Comment by apeatling** [comment]
@Automattic/lannister Might be something to pick up when looking at lead gen.

**Comment by gwwar** [comment]
@apeatling I believe this was already implemented by @Automattic/tanooki in p3Ex-32O-p2

**Pull request closed without merging**